api/tests/opentrons/protocol_engine/state/test_pipette_store.py

```diff
@@ -416,26 +422,36 @@ def test_heater_shaker_command_without_movement(
             None,
         ),
         (
-            create_move_labware_off_deck_command(labware_id="non-matching-labware-id"),
+            create_move_labware_command(
+                labware_id="non-matching-labware-id",
+                strategy="manualMoveWithoutPause",
```

Comment:
```suggestion
                strategy="manualMoveWithPause",
```

I know `manualMoveWithoutPause` is equivalent to what this test case was doing before this PR, but `manualMoveWithPause` is consistent with the other test cases, so maybe it improves readability? Right now, it's not clear if the "special thing" in this test case is `OFF_DECK_LOCATION`, or `manualMoveWithPause`, or both.

Reply:
good point